modules/cpmso-config-options.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cpmso-config-prefix_{context}"]
+= Adding a custom prefix to control plane machine names
+
+You can customize the prefix of machine names created by the control plane machine set.
+This can be done by editing the `ControlPlaneMachineSet` custom resource (CR)
+
+.Procedure
+
+. Edit the `ControlPlaneMachineSet` CR by running the following command:
++
+[source,terminal]
+----
+$ oc edit controlplanemachineset.machine.openshift.io cluster \
+  -n openshift-machine-api
+----
+
+. Edit the `.spec.machineNamePrefix` field of the `ControlPlaneMachineSet` CR:
++
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1
+kind: ControlPlaneMachineSet
+metadata:
+  name: cluster
+  namespace: openshift-machine-api
+spec:
+  machineNamePrefix: <machine_prefix>
+# ...
+----
++
+where `<machine_prefix>` specifies a prefix name that follows the requirements for a lowercase RFC 1123 subdomain.
++
+[IMPORTANT]
+====
+A lowercase RFC 1123 subdomain must consist of only lowercase alphanumeric characters, hyphens ('-'), and periods ('.').
+Each block, separated by periods, must start and end with an alphanumeric character.
+Hyphens are not allowed at the start or end of a block, and consecutive periods are not permitted.
+====
+
+. Save your changes.
+
+.Next steps
+
+* Changing the value of the `machineNamePrefix` parameter does not cause an automatic update for clusters that use the default `RollingUpdate` update strategy.
```

Comment:
How about adding `only` here? like 

Only changing the value of the `machineNamePrefix` parameter does not cause an automatic update for clusters that use the default `RollingUpdate` update strategy.

Because if there are other changes along with the `machineNamePrefix` parameter, then it will cause an automatic update for clusters that use the default `RollingUpdate` update strategy. New master machines' name will be prefixed with the value you set for `machineNamePrefix`.

Reply:
good point! :+1:

Reply:
Great, updating this to specify using the word "only" here